### PR TITLE
Add build vm assets

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
Move script build-vm-assets to the image.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/templates/macOS-10.13.json [hunk 1]
```diff
@@ -43,6 +43,11 @@
             "type": "shell",
             "inline": "mkdir ~/image-generation"
         },
+        {
+            "type": "file",
+            "source": "./provision/assets",
+            "destination": "~/image-generation/"
+        },
```
Comment (AlenaSviridenko) [inline images/macos/templates/macOS-10.13.json:50]: Do we need these assets on High Sierra?

Comment (dmitry-shibanov) [reply images/macos/templates/macOS-10.13.json:50]: I think we need it for High Sierra.

**Review by maxim-lobanov — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

File: images/macos/provision/configuration/finalize-vm.sh [hunk 2]
```diff
@@ -16,6 +16,14 @@ fi
 # Put documentation to $HOME root
 cp $HOME/image-generation/output/software-report/systeminfo.txt $HOME/image-generation/output/software-report/systeminfo.md $HOME/
 
+# Put build vm assets scripts to proper directory
+sudo mkdir -p /usr/local/opt/$USER/scripts
```
Comment (miketimofeev) [inline images/macos/provision/configuration/finalize-vm.sh:20]: why we need sudo here?

File: images/macos/provision/configuration/finalize-vm.sh [hunk 3]
```diff
@@ -16,6 +16,14 @@ fi
...
+sudo chmod ugo+rwx /usr/local/opt/$USER/scripts
+sudo chown $USER:admin /usr/local/opt/$USER
```
[the remark this answers is not in the document]
Comment (dmitry-shibanov) [reply images/macos/provision/configuration/finalize-vm.sh]: Removed this lines.

File: images/macos/provision/configuration/finalize-vm.sh (hunk 2 above)
Comment (dmitry-shibanov) [reply images/macos/provision/configuration/finalize-vm.sh]: Removed sudo

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**